packages/perseus-editor/src/components/color-select.tsx

```diff
@@ -22,34 +23,36 @@ type Props = {
 const ColorSelect = (props: Props) => {
     const {selectedValue, style, onChange} = props;
 
-    const ids = useUniqueIdWithMock();
-    const id = ids.get("color-select");
-
     return (
         <View style={[styles.row, style]}>
-            <LabelMedium htmlFor={id} style={styles.label} tag="label">
-                color
-            </LabelMedium>
-            <SingleSelect
-                id={id}
-                selectedValue={selectedValue}
-                onChange={onChange}
-                // Placeholder is required, but never gets used.
-                placeholder=""
-            >
-                {possibleColors.map((colorName) => (
-                    <OptionItem
-                        key={colorName}
-                        value={colorName}
-                        label={colorName}
-                        leftAccessory={
-                            <ColorSwatch color={colorName} decorative={true} />
-                        }
+            <LabelMedium tag="label">
+                <View style={styles.row}>
```

Comment:
I tried to make this work without this extra `View` here, but adding `style={styles.row}` directly on the `LabelMedium` didn't do anything... open to other suggestions if anyone knows how I can avoid this extra `View`.

Reply:
Looks like `styles.row` sets the flex direction, but does not set `display: flex` on the element. `View`s are `display: flex` by default, but I guess `LabelMedium`s aren't.

I'm guessing if you add `display: flex` to `styles.row`, it will work.

Reply:
You're right! I never would have guessed that a Wonder Blocks element doesn't have `display: "flex"`.